app/js/components/manageApps/ManageApps.jsx

```diff
@@ -795,12 +796,36 @@ export default class ManageApps extends React.Component {
     const disableUploadElements = (uploadStatus > 0) ? true : false;
     disableUploadElements ? document.body.className = 'loading' : document.body.className = '';
 
-    return (
-      <div>
+    
+
+  const buttonsInstance = (
+      <DropdownButton title="Support" id="startall-modules-btn">
+        <MenuItem 
+          eventKey="1">
+          <span>
+            <Link to="/manageSettings" className="manage-settings-button support-dropdown">
+              Settings
+            </Link>
+          </span>
+        </MenuItem>
+        <MenuItem
+          eventKey="2">
+          <span>
+            <Link to="/help" className="manage-settings-button support-dropdown">
+              Help Guide
+            </Link>
+          </span>
+        </MenuItem>
+      </DropdownButton>
+    );
+ return (
```

Comment:
For the support drop down, wouldn't it have been better to have left the settings icon because the button name support sounds ambiguous to hold both settings and help guide. You should also add padding bottom to the drop down. Also the bright blue color scheme of the links in the drop down should be changed to the same color scheme as that used in the rest of the page.

Reply:
@malmike For better UI I removed the settings icon so as to maintain uniformity for both links